src/main/resources/application-local.yml

```diff
@@ -30,8 +30,14 @@ hmpps.sqs:
       dlqMaxReceiveCount: 1
     cprdeliusoffendereventsqueue:
       queueName: cpr_delius_offender_events_queue
+      dlqName: cpr_delius_offender_events_queue_dlq
       subscribeTopicId: domainevents
-      subscribeFilter: '{"eventType":[ "probation-case.engagement.created", "prisoner-offender-search.prisoner.created", "prisoner-offender-search.prisoner.updated"] }'
+      subscribeFilter: '{"eventType":[ "probation-case.engagement.created"] }'
```

Comment:
should have the other offender events we added this morning in this subscription (not related to this change I know)